test_util/aws.py

```diff
@@ -170,30 +195,14 @@ def empty_and_delete_s3_bucket_from_stack(self):
         bucket.delete()
         log.info('Delete successfully triggered for {}'.format(self.stack.stack_name))
 
-    def get_master_ips(self, state_list=None):
-        instances = self.get_group_instances(['MasterServerGroup'], state_list)
-        return instances_to_hosts(instances)
-
-    def get_public_agent_ips(self, state_list=None):
-        instances = self.get_group_instances(['PublicSlaveServerGroup'], state_list)
-        return instances_to_hosts(instances)
+    def get_master_ips(self):
+        return instances_to_hosts(self.get_auto_scaling_instances('MasterServerGroup'))
```

Comment:
should these be filtering for running like https://github.com/dcos/dcos/pull/933/files#diff-0ba5c9c4483f599539d64bc5a299545dR37 does?